SU-HSDO/suhumsci
# feat(STN-344):  Traditional Theme Sweeping Element Styles

# READY FOR REVIEW

## Summary
Adds and edits Traditional theme sweeping elements.

👍 make sure I didn't break anything in Colorful.

Figma designs: https://www.figma.com/file/cySH8EVRf71kTj866OQmfM/%F0%9F%8E%A8--Traditional-Theme---Team-Library?node-id=25%3A4

## Steps to Test
1. Ensure that `npm run test` passes in humsci_basic
2. Ensure the following are true:

- [x] More Link style matches the figma designs on these pages:
   - http://economics.suhumsci.loc/qa/qa-cards
   - http://economics.suhumsci.loc/qa/qa-real-econ-content
- [x] If you go to a node detail page, and use the layout tool to add a class to the Title of the page: `hb-title` it doesn't change anything, or add a colored above the title.
- [x] On the node detail page, add the class `ht-serif` to a field to see the serif Traditional theme style applied.
- [x] Ensure the headings in the local footer use a sans-serif style.
- [x] Ensure that the form elements on the following pages, match Figma:
   - http://economics.suhumsci.loc/courses
  - Search bar on any page in the header

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
